dist/assets/dotgov-accordion.min.js

```diff
@@ -95,7 +95,7 @@ var BcAccordion =
 
```

Comment:
I need to make sure we add these .min files to the ignore list. There isnt any reason to store these in source since we generate them on the fly. I'll make a note of this.